updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc

```diff
@@ -114,7 +114,7 @@ and must be no more than 63 characters`, try creating the Update Service with a
 ====
 
 // Configuring the Cluster Version Operator (CVO)
-include::modules/update-service-configure-cvo.adoc[leveloffset=+3]
+include::modules/update-service-configure-cvo.adoc[leveloffset=+1]
```

Comment:
Out of scope but I'm bumping up the level of this section since it is its own part of the process - otherwise it looks like "configuring the CVO" is a subsection of "Creating an OpenShift Update Service application by using the CLI"